app/helpers/application_helper.rb

```diff
@@ -415,4 +415,8 @@ def edit_inline(object, property, options = {})
       h(value)
     end
   end
+
+  def table_css_classes
```

Comment:
I think that's out of scope for this PR but if you start working on table helper which @ohadlevy suggests it would be nice to bring full solution that would allow specifying columns, adding columns from plugins or maybe incorporate something [existing](https://www.ruby-toolbox.com/categories/Table_Builders). If we start building small helpers, it might end up as it is with form helpers.

sorry for OT :-)